pgmpy/tests/test_factors/test_Factor.py

```diff
@@ -99,6 +118,17 @@ def test_marginalize(self):
         self.phi1.marginalize(['x3'])
         np_test.assert_array_equal(self.phi1.values, np.array(66))
 
+    def test_marginalize_hashable(self):
```

Comment:
Rename this function, even test_marginalize1 or test_marginalize2 something like that.